cxf-interceptors/src/main/java/pl/touk/cxf/interceptors/threadname/policy/ThreadNamePrefixPolicy.java

```diff
@@ -0,0 +1,18 @@
+package pl.touk.cxf.interceptors.threadname.policy;
+
+import lombok.AllArgsConstructor;
+import pl.touk.cxf.interceptors.threadname.ThreadNamePolicy;
+
+@AllArgsConstructor
+public class ThreadNamePrefixPolicy implements ThreadNamePolicy {
+
+    private final String threadNamePrefix;
```

Comment:
Remove final, add setter. It would be easier to configure using XML (e.g. blueprint).

Reply:
It can be configure via constructor injection as well.